### DISCUSSION TITLE
after setup source code, the webserver show "Please make sure to build the frontend in static/ directory and restart the server"

### DISCUSSION THREAD

**Discussion by gi112233** [apache/airflow#40808] [Q&A]
I am learning and trying to add some customization features, such as determining memory conditions in executor. When I was done with that, I repackaged the project and then reinstalled it.

After restarting the service, I noticed that my web interface became very strange and found this message "Please make sure to build the frontend in static/ directory and restart the server"

Is there anything I did wrong or missed

The source code version I use is 2.6.3

**Comment by jedcunningham** [comment]
If you are installing Airflow from source, you will need to build the frontend assets. Instructions on how to do it are in the [INSTALL file](https://github.com/apache/airflow/blob/afde88ac336b34c89654b7f4e2f23fae64863134/INSTALL#L336).
